x-pack/platform/plugins/shared/fleet/common/services/package_helpers.test.ts

```diff
@@ -119,3 +120,41 @@ describe('getRootIntegrations', () => {
     expect(res).toEqual([]);
   });
 });
+
+describe('hasInstallServersInputs', () => {
+  it('should return true if any package policy has fleet-server input', () => {
+    const res = hasInstallServersInputs([
+      {
+        inputs: [{ type: 'fleet-server' }],
```

Comment:
Looking at the tests I thought of policies that have "simplified" inputs, i.e. they are not an array but a record. Should we test for those as well?

Reply:
Where this function is used we only use `PackagePolicy`, I think the simplified type is not relevant here.